src/libraries/System.Net.Security/src/System/Net/Security/SslCertificateTrust.cs

```diff
@@ -0,0 +1,66 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Versioning;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net.Security
+{
+    public sealed partial class SslCertificateTrust
+    {
+        internal X509Store? _store;
+        internal X509Certificate2Collection? _trustList;
+        internal bool _sendTrustInHandshake;
+
+        public static SslCertificateTrust CreateForX509Store(
```

Comment:
Nit: We usually either put the params all on one line, or we put them on separate lines with one level of indentation off the original function declaration. Seems like we should do that here too, and below.

Reply:
updated.